### DISCUSSION TITLE
Links to videos and presentations on (or using) Makie

### DISCUSSION THREAD

**Discussion by asinghvi17** [JuliaPlots/Makie.jl#4239] [Ideas]
It would be neat if there were a page in the docs which links to Makie tutorials and videos out in the wild!  Right now I think you have to use Google to find many of them - there's no link from e.g. the docs to the Julia Data Science tutorial which was quite neat.

**Comment by bperard** [comment]
I think I could help out with this. Would you want these other videos added into a subsection (and if so, is there a preferred name; community videos/presentations?), or with the two other videos on your tutorials page?

**Comment by asinghvi17** [comment]
Thanks!  This would ideally be a separate section, I think, since there are a few talks about e.g. the architecture of Makie which aren't traditional tutorials (and a lot more will be coming in a couple of weeks from MakieCon).

A new page titled community videos sounds good!  Alternatively, we could call it "Makie in the wild" or something, and then have it be a list of all references to Makie.  What sounds better to you?

**Comment by bperard** [comment]
I had both of those ideas in mind from your description; I might be able to make them both work with one used as a subsection. I'll gather the videos and see what looks best.
